models/netsuite/netsuite__balance_sheet.sql

```diff
@@ -54,14 +54,10 @@ balance_sheet as (
     case
       when lower(accounts.is_balancesheet) = 'f' then null
       else accounts.account_number
-        end as account_number,
+        end as account_number
     
     --The below script allows for accounts table pass through columns.
-    {% if var('accounts_pass_through_columns') %}
-    
-    accounts.{{ var('accounts_pass_through_columns') | join (", accounts.")}} ,
-
-    {% endif %}
+    {{ fivetran_utils.persist_pass_through_columns('accounts_pass_through_columns', identifier='accounts') }},
```

Comment:
@fivetran-sheringuyen @fivetran-catfritz I would like to get your input here as well.

Reply:
Hey Joe, sorry for the delay in response. I was able to get dbt to run with the changes you proposed, but why would that be different than all the other pass-through column calls?

Reply:
Hi @kchiodo this one is a bit different because we need to pass these pasthrough fields in as `null` in order for a union to work. The underlying macro that was added to this update which handles these passthrough fields was not developed to handle this `null` functionality. 

This may be something we can update within the deeper fivetran_util at a later date, but for the meantime this workaround worked as intended! If you want to add this to your PR, we can move forward with working to incorporate your changes into the next Netsuite release!